velox/type/tests/TypeTest.cpp

```diff
@@ -134,6 +134,9 @@ TEST(TypeTest, shortDecimal) {
   } catch (const VeloxRuntimeError& e) {
     EXPECT_EQ("(19 vs. 18)", e.message());
   }
+  EXPECT_TRUE(SHORT_DECIMAL(10, 5)->kindEquals(SHORT_DECIMAL(10, 5)));
```

Comment:
Not related to this PR, but in the above tests we should try to construct a SHORT_DECIMAL with negative precision and scale and throw exception in those cases.